frontends/web/src/components/terms/bitsurance-terms.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { ChangeEvent } from 'react';
+import { Button, Checkbox } from '../forms';
+import { setConfig } from '../../utils/config';
+import { A } from '../anchor/anchor';
+import style from './terms.module.css';
+
+type TProps = {
+  onAgreedTerms: () => void;
+}
+
+export const BitsuranceTerms = ({ onAgreedTerms }: TProps) => {
+  const { t } = useTranslation();
+  const handleSkipDisclaimer = (e: ChangeEvent<HTMLInputElement>) => {
+    setConfig({ frontend: { skipBitsuranceDisclaimer: e.target.checked } });
+  };
+
+  return (
+    <div className={style.disclaimerContainer}>
+      <div className={style.disclaimer}>
+        <p>{t('bitsurance.terms.text1')}</p>
+        <p>{t('bitsurance.terms.text2')}</p>
+        <p>{t('bitsurance.terms.text3')}</p>
+        <p>{t('bitsurance.terms.text4')}</p>
+        <p>
+          {t('bitsurance.terms.text5')}
+          {' '}
+          <A href="https://www.bitsurance.eu/en/dataprotection">
```

Comment:
Link to https://www.bitsurance.eu/datenschutz/ would be nice to have too.

Reply:
Good point, actually! Thanks

Reply:
We used to put it in locize but it got messy quickly and afaik we started putting having links in JSX directly with language switch. I guess German-first is ok in this case (their english page is not ready yet).